packages/algob/src/lib/deployer.ts

```diff
@@ -18,18 +22,16 @@ export class AlgobDeployerImpl implements AlgobDeployer {
   private readonly cpData: CheckpointRepo;
   private readonly loadedAsaDefs: ASADefs;
   private readonly algoClient: AlgoDeployClient;
+  readonly accounts: Accounts;
 
   constructor (
-    runtimeEnv: AlgobRuntimeEnv, cpData: CheckpointRepo, asaDefs: ASADefs, algoSdk: AlgoDeployClient
+    runtimeEnv: AlgobRuntimeEnv, cpData: CheckpointRepo, asaDefs: ASADefs, algoClient: AlgoDeployClient
   ) {
     this.runtimeEnv = runtimeEnv;
     this.cpData = cpData;
     this.loadedAsaDefs = asaDefs;
-    this.algoClient = algoSdk;
-  }
-
-  get accounts (): Account[] {
-    return this.runtimeEnv.network.config.accounts;
+    this.algoClient = algoClient;
+    this.accounts = mkAccountIndex(runtimeEnv.network.config.accounts);
```

Comment:
this has a wrong type. Please check the spec doc. I intentionally mentioned in the chat to make sure we will not go again in the loops because we are diverging from spec without previous agreement. The spec is there for a reason to make sure that everyone is on the same page.

Reply:
Ok, didn't know that it was needed to have both, not only one.

Reply:
accounts: Account[];
  accountsByName: Accounts;

Reply:
Updated. Now it contains both vars.